lib/galaxy/jobs/runners/slurm.py

```diff
@@ -201,3 +202,34 @@ def __check_memory_limit(self, efile_path):
             log.exception('Error reading end of %s:', efile_path)
 
         return False
+
+
+def _remove_spurious_top_lines(rfh, ajs, maxlines=2):
+    bad = []
+    putback = None
+    i = 0
+    while i < maxlines:
+        line = rfh.readline()
+        log.debug('#### checking line: %s', line)
```

Comment:
Maybe `log.trace()` here?

Reply:
Whoops, that wasn't even supposed to be in there, but I like the idea.